v6/v6_interface.ml

```diff
@@ -22,54 +25,56 @@ let default_path = ref (Filename.concat default_sockets_dir service_name)
 let uri () = "file:" ^ !default_path
 let json_path = "/var/xapi/v6.json"
 let xml_path = "/var/xapi/v6"
+
 (* --- global types used in most API calls --- *)
 
-type debug_info = string
 (** Uninterpreted/sanitised string associated with operation *)
+type debug_info = string
 [@@deriving rpcty]
 
+(** Record representing software edition *)
 type edition =
-{
-  name : string ;
-  (** Name of edition, this will be passed to apply_edition *)
-  official_title : string ;
-  (** Marketing title used to advertise edition *)
-  code : string ;
-  (** Abbreviated form of name, used to show up in logs and on command line *)
-  order : int ;
-  (** Number indicating ordering among other editions
-      with low numbers corresponding to editions with fewer features, and vice versa *)
-}
+  {
+    ed_title : string ;
```

Comment:
Do we really need these prefixes? I generally prefer to wrap a type in a module instead. In this case, this type is already in module `V6_interface` and the field names don't seem to conflict with anything else in here?

Reply:
They were originally necessary to prevent namespace clashes in large functions with an indiscriminate `let open V6_interface in `, but now all functions using them are wrapped in the module namespace, so I can remove the prefixes.